src/image/ImageAnnotator.scss

```diff
@@ -15,6 +15,6 @@
     // transform applied to each of the managers' react mount element to account for
     // rotation
     .ba-Layer--popup {
-        z-index: 1;
+        z-index: 4;
```

Comment:
do we need this if we have the baseannotator.scss changes?

Reply:
I believe so, because this is more specific than the styling in BaseAnnotator; so if we don't do this, this'll override the z-index styling for image files which was the issue that QA was seeing earlier